rest/routing.go

```diff
@@ -357,6 +363,14 @@ func CreateMetricRouter(sc *ServerContext) *mux.Router {
 	return r
 }
 
+func CreateDiagnosticRouter(sc *ServerContext) *mux.Router {
```

Comment:
You'll want to create a corresponding API docs in `docs/api/diagnostic.yml`.

I test this with:

```
npx @redocly/cli preview-docs api/docs/admin.yml
```